src/main/java/aztech/modern_industrialization/MITooltips.java

```diff
@@ -386,6 +387,17 @@ public Component parse(Double ratio) {
             MIText.ConfigCardHelpItems5,
             MIText.ConfigCardHelpClear);
 
+    public static final TooltipAttachment MACHINE_CASING_VOLTAGE = TooltipAttachment.of(
+            (itemStack, item) -> {
+                var itemKey = BuiltInRegistries.ITEM.getKey(item);
```

Comment:
This should use the static method in CasingComponent.

Reply:
sweet, will do